### PR TITLE
Refactor DBUser attribute

### PR DESCRIPTION
**Pull request by ektabj**
# Description of the issue
Refactor DBUser attribute

# Description of changes
Refactor DBUser attribute

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
Ran below commands, succeeded without errors.
- `make build`

# Requirements
_Before commit the code, please do the following steps._
1. Run `make fmt` and `make fmt-sh`
2. Run `make lint`

### CODE REVIEW

File: plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go [hunk 1]
```diff
@@ -40,7 +44,6 @@ var attributesRenamingForMetric = map[string]string{
 	attr.AWSRemoteTarget:             common.MetricAttributeRemoteResourceIdentifier,
 	attr.AWSRemoteResourceIdentifier: common.MetricAttributeRemoteResourceIdentifier,
 	attr.AWSRemoteResourceType:       common.MetricAttributeRemoteResourceType,
-	attr.AWSRemoteDbUser:             common.MetricAttributeRemoteDbUser,
```
Comment (bjrara) [inline plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go:43]: `attributesRenamingForMetric` is created to maintain the list of attributes to be renamed for attributes in EMF logs, so keeping `AttributeRemoteDbUser` here is ok. But now I see how much confusion it caused from the current naming.

**Comment by bjrara** [comment]
Hi @ektabj I submitted a new PR to refactor the Application Signals attributes. It covers the renaming of RemoteDbUser. PTAL https://github.com/aws/amazon-cloudwatch-agent/pull/1248

File: plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go (hunk 1 above)
Comment (ektabj) [reply plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go:43]: In that case, I'll close this pull request if it's fine with you?

Comment (bjrara) [reply plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go:43]: Yes. Thank you!

**Pull request closed without merging**